app/main.py

```diff
@@ -1,8 +1,45 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
```

Comment:
After opening the parenthesis, write all the arguments on a new line starting from the first one, also write the closing parenthesis on a new line